### PR TITLE
Fix azure config

### PR DESCRIPTION
**Pull request by casparvl**

### CODE REVIEW

File: config/azure_mc.py [hunk 1]
```diff
@@ -33,11 +33,30 @@
...
                     'access': ['--partition=x86-64-amd-zen4-node', '--export=NONE'],
                     'descr': 'Zen4, 16 cores, 30 GB',
-                },
+                    'prepare_cmds': [
+                        'export EESSI_SOFTWARE_SUBDIR_OVERRIDE=x86_64/amd/zen4',
+                        common_eessi_init(),
+                        # Required when using srun as launcher with --export=NONE in partition access, in order to ensure job
+                        # steps inherit environment. It doesn't hurt to define this even if srun is not used
+                        'export SLURM_EXPORT_ENV=ALL'
+                    ]
```
Comment (smoors) [inline config/azure_mc.py:42]:
```suggestion
                    ],
```

**Comment by smoors** [comment]
@casparvl fixing a typo uncovered a bunch of style issues

>./config/azure_mc.py:39:121: E501 line too long (125 > 120 characters)
./config/azure_mc.py:44:24: E121 continuation line under-indented for hanging indent
./config/azure_mc.py:45:20: E122 continuation line missing indentation or outdented
./config/azure_mc.py:46:16: E122 continuation line missing indentation or outdented
./config/azure_mc.py:53:121: E501 line too long (125 > 120 characters)
./config/azure_mc.py:58:24: E121 continuation line under-indented for hanging indent
./config/azure_mc.py:59:20: E122 continuation line missing indentation or outdented

**Comment by casparvl** [comment]
Style issues are my trademark :P I'll fix them :)

**Review by smoors — APPROVED** [review]
lgtm

**Pull request merged**